### ISSUE TITLE
Navigation and status bars appear for a short while when switching activities (via Intents) despite being in immersive sticky mode

### ISSUE THREAD

**Issue by liweiyap** [liweiyap/narradir-android#2]
Exactly the same problem encountered here:
 *  https://stackoverflow.com/questions/31198923/preventing-android-navigation-and-action-bars-from-appearing-when-switching-acti
 * https://stackoverflow.com/questions/34394916/why-navigation-bar-is-appearing-in-fullscreen-apps-when-clicked-on-popup-menu

This, despite update to [`FullScreenPortraitActivity`](https://github.com/liweiyap/narradir-android/blob/main/app/src/main/java/com/liweiyap/narradir/utils/FullScreenPortraitActivity.java) in commit 014d119dcb7f674bd0e92d8fa0a5171cbee51f5d

**Comment by liweiyap** [comment]
Related issue in #3

**Comment by liweiyap** [comment]
Fixed the related issue #3 but this issue remains unresolved.

**Comment by liweiyap** [comment]
Maybe replace Activities with Fragments to solve this issue.

**Comment by liweiyap** [comment]
Closing this because I haven't been able to reproduce it for a long time. I did fix a related bug in commit fa1fe5b63f574dc959b2dbd6aaeb696a8d0b1bf0 though.

Nonetheless, it is important to note that this does not mean that we shouldn't, at some point in the future, rewrite the code base to convert our existing Activities into Fragments, so as to shift some code into a ViewModel and reduce code repetition.
